cumulusci.yml

```diff
@@ -37,8 +37,8 @@ tasks:
         options:
             path: unpackaged/config/dev_delete
 
-    execute_install_apex:
-        description: Runs most of the install script methods from STG_InstallScript class
+    execute_dev_apex:
```

Comment:
@jstvz I renamed it to dev_apex since it feels like it is running apex for dev orgs. We can change it back if you prefer install_apex thou.

Reply:
It is doing most of the install script though, of a new install, so the old name does sort of make sense. It's doing lines 44-63 of STG_InstallScript.cls, minus the scheduling of the scheduled jobs and running the batch email script.

(I don't have a strong opinion either way)

Reply:
Fair enough, qa config is also running this thus I don't think it makes sense to call it dev_apex either...Ok I will change it back.